src/test/java/org/wildfly/security/ssl/MechanismDatabaseTest.java

```diff
@@ -43,4 +45,21 @@ public void testEdhDheMapping() {
         System.out.println(entry);
     }
 
+    @Test
+    public void testAllJdkCipherSuitesMapping() {
+        final MechanismDatabase mechanismDatabase = MechanismDatabase.getInstance();
+
+        SSLServerSocketFactory ssf = (SSLServerSocketFactory)SSLServerSocketFactory.getDefault();
+        String[] supportedCipherSuites = ssf.getSupportedCipherSuites();
+
+        StringBuffer unknownCipherSuites = new StringBuffer("");
+        for (int i = 0; i < supportedCipherSuites.length; i++) {
+            System.out.println("Trying " + supportedCipherSuites[i]);
```

Comment:
If you drop this message and instead print the list of unsupported suites to stdout, we can see it directly from GitHub and its emails instead of having to visit CI to see the problem.